cmake/SetRevision.cmake

```diff
@@ -0,0 +1,22 @@
+macro(SetRevision major minor patch dev)
+
+    set(${PROJECT_NAME}_VERSION_MAJOR ${major} CACHE STRING "OpenMEEG-superproject major version number.")
+    mark_as_advanced(${PROJECT_NAME}_VERSION_MAJOR)
+
+    set(${PROJECT_NAME}_VERSION_MINOR ${minor} CACHE STRING "OpenMEEG-superproject minor version number.")
+    mark_as_advanced(${PROJECT_NAME}_VERSION_MINOR)
+
+    set(${PROJECT_NAME}_VERSION_PATCH ${patch} CACHE STRING "OpenMEEG-superproject build version number.")
+    mark_as_advanced(${PROJECT_NAME}_VERSION_PATCH)
+
+    set(${PROJECT_NAME}_VERSION_TWEAK ${dev} CACHE STRING "OpenMEEG-superproject development marker.")
```

Comment:
I would try to name it OpenMEEG for both and allow to build from root or
OpenMEEG folder